src/test/java/net/testudobank/tests/MvcControllerIntegTest.java

```diff
@@ -239,6 +240,67 @@ public void testWithdrawTriggersOverdraft() throws SQLException, ScriptException
     MvcControllerIntegTestHelpers.checkTransactionLog(customer1TransactionLog, timeWhenWithdrawRequestSent, CUSTOMER1_ID, MvcController.TRANSACTION_HISTORY_WITHDRAW_ACTION, CUSTOMER1_AMOUNT_TO_WITHDRAW_IN_PENNIES);
   }
 
+/**
+   * 
+   * The customer will be given an initial balance of $100 and will withdraw $1099.
+   * This will test the scenario where the overdraft balance itself, which is $100 - $1099 = -$999,
+   * is a valid overdraft balance, but once the 2% interest rate is applied, will cross the limit
+   * of $1000. 
+   * 
+   * Checks to make sure that the customer's balance stays the same as before due to a failed 
+   * withdraw request, and checks that the TransactionHistory table is empty.
+   * 
+   * @throws SQLException
+   * @throws ScriptException
+   */
+
+  @Test
+  public void testWithdrawOverdraftOverage() throws SQLException, ScriptException { 
```

Comment:
Change the name of this method to `testWithdrawOverdraftLimitExceeded()` to add more clarity. `Overage` is kind of vague.